frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -199,7 +199,7 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
             disabled={false}
           />
           <AddressForm
-            onGeocoderChange={handleGeocoderChanges}
+            onGeocoderChange={noop}
```

Comment:
the address form in the data section shouldn't trigger searches, only the search address should.